src/deckgl/raster-layer/raster-layer.js

```diff
@@ -98,7 +99,8 @@ export default class RasterLayer extends BitmapLayer {
 
   updateState({props, oldProps, changeFlags}) {
     // setup model first
-    if (changeFlags.extensionsChanged) {
+    const modulesChanged = props && props.modules && !isEqual(props.modules, oldProps.modules);
```

Comment:
I think you need to check that `oldProps.modules` exists too. On a new layer I think `oldProps` can maybe occasionally be null, if I remember correctly.